### DISCUSSION TITLE
Creating an episode.

### DISCUSSION THREAD

**Discussion by teteva** [statamic/starter-kit-podcaster#10] [Q&A]
Why is it not possible to place a link to an audio file when creating an episode, instead of downloading an audio file. For example, I store audio files on Google Cloud.
![Image 17 04 2023 at 10 24 PM](https://user-images.githubusercontent.com/16236531/232590470-c6179ddd-3727-4c7b-b474-6fccf7da4c58.jpeg)

**Comment by jackmcdade** [comment]
You certainly could! Replace that field with a `link` fieldtype in the Blueprint and voila! Done.
